### PR TITLE
Add basic form and search parameters

### PR DESCRIPTION
**Pull request by powazek**

### CODE REVIEW

File: client/death/FormPage.js [hunk 1]
```diff
@@ -238,6 +256,7 @@ export default class IndexPage extends React.Component {
 
             <div className="txt">
               <label htmlFor="text" className="txt-l">Dog Age (Years)</label>
+              <style textAlign="right" />
```
Comment (fionawhim) [inline client/death/FormPage.js]: What are you trying to achieve here? I think you probably want `style={{textAlign: 'right'}}` on the `<input />` below.

File: client/death/FormPage.js [hunk 2]
```diff
@@ -102,7 +120,7 @@ export default class IndexPage extends React.Component {
 
   render() {
     const { results, cart } = this.props;
-    const { query } = this.state;
+    //const { query } = this.state;
```
Comment (fionawhim) [inline client/death/FormPage.js]: Just remove query from here and from the state.

File: client/death/FormPage.js [hunk 3]
```diff
@@ -238,6 +256,7 @@ export default class IndexPage extends React.Component {
 
             <div className="txt">
               <label htmlFor="text" className="txt-l">Dog Age (Years)</label>
+              <style textAlign="right" />
               <input
                 id="text"
```
Comment (fionawhim) [inline client/death/FormPage.js]: You’ll need to update these `id`s so that they're unique.

File: client/death/FormPage.js [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+      breed1: '',
+      breed2: '',
+      sex: '',
+      rabiesIssued: '',
+      rabiesExpire: '',
+      comments: '',
+    };
+  }
+
+  handleQueryChange = (ev: SyntheticInputEvent) => {
```
Comment (fionawhim) [inline client/death/FormPage.js]: can remove this method.

**Review by fionawhim — CHANGES_REQUESTED** [review]
Just a few little things. After this PR is in, I'd recommend doing a change from "death" to "dog" for all of the directories and paths, and put that up as a PR of its own. Renames are always best to handle on their own.

File: client/death/FormPage.js (hunk 1 above)
Comment (powazek) [reply client/death/FormPage.js]: I was just testing with this to see if it changed position, I'll remove it

File: client/death/FormPage.js (hunk 2 above)
Comment (powazek) [reply client/death/FormPage.js]: Done!

File: client/death/FormPage.js (hunk 4 above)
Comment (powazek) [reply client/death/FormPage.js]: Done

File: client/death/FormPage.js [hunk 5]
```diff
@@ -146,7 +116,7 @@ export default class IndexPage extends React.Component {
             <div className="txt">
               <label htmlFor="text" className="txt-l">Owner First Name</label>
               <input
-                id="text"
+                id="lookup-firstname"
```
Comment (fionawhim) [inline client/death/FormPage.js]: Be consistent w/ casing. `firstName`

File: client/death/FormPage.js [hunk 6]
```diff
@@ -146,7 +116,7 @@ export default class IndexPage extends React.Component {
             <div className="txt">
               <label htmlFor="text" className="txt-l">Owner First Name</label>
```
Comment (fionawhim) [inline client/death/FormPage.js]: `htmlFor` attributes need to match the `id`s of the `input` elements, here and below.

File: client/death/FormPage.js (hunk 1 above)
Comment (fionawhim) [reply client/death/FormPage.js]: Remember to remove.

File: client/death/FormPage.js [hunk 7]
```diff
@@ -258,21 +233,21 @@ export default class IndexPage extends React.Component {
...
             </div>
 
             <div className="sel">
-              <label htmlFor="language" className="sel-l">
+              <label htmlFor="breed" className="sel-l">
                 Dog Breed
               </label>
               <div className="sel-c">
-                <select name="language" id="language" className="sel-f">
+                <select name="breed" id="lookup-breed" className="sel-f">
```
Comment (fionawhim) [inline client/death/FormPage.js]: Should have `value={this.state.breed}`

File: client/death/FormPage.js [hunk 8]
```diff
@@ -294,11 +269,11 @@ export default class IndexPage extends React.Component {
             </div>
 
             <div className="sel">
-              <label htmlFor="language" className="sel-l">
+              <label htmlFor="color" className="sel-l">
                 Primary Dog Color
               </label>
               <div className="sel-c">
-                <select name="language" id="language" className="sel-f">
+                <select name="color" id="lookup-color" className="sel-f">
```
Comment (fionawhim) [inline client/death/FormPage.js]: `value` attribute here, too

File: client/death/FormPage.js [hunk 9]
```diff
@@ -310,21 +285,47 @@ export default class IndexPage extends React.Component {
             </div>
 
             <div className="sel">
-              <label htmlFor="language" className="sel-l">
+              <label htmlFor="sex" className="sel-l">
                 Dog Sex
               </label>
               <div className="sel-c">
-                <select name="language" id="language" className="sel-f">
+                <select name="sex" id="lookup-sex" className="sel-f">
```
Comment (fionawhim) [inline client/death/FormPage.js]: `value`

**Review by fionawhim — CHANGES_REQUESTED** [review]

File: client/death/FormPage.js [hunk 10]
```diff
@@ -114,7 +114,9 @@ export default class IndexPage extends React.Component {
             <input name="utf8" type="hidden" value="✓" />
 
             <div className="txt">
-              <label htmlFor="text" className="txt-l">Owner First Name</label>
+              <label htmlFor="lookup-firstName" className="txt-l">
+                Owner First Name
+              </label>
               <input
                 id="lookup-firstname"
```
Comment (fionawhim) [inline client/death/FormPage.js]: Need to update this ID to match case.

File: client/death/FormPage.js (hunk 1 above)
Comment (fionawhim) [reply client/death/FormPage.js]: Still there?

**Review by fionawhim — APPROVED** [review]
Fix these two tiny, things, then you're OK to merge this.

**Comment by fionawhim** [comment]
If tests pass, that is.

File: client/death/FormPage.js (hunk 10 above)
Comment (powazek) [reply client/death/FormPage.js]: Done! What is the second tiny thing?

**Pull request merged**